src/index.html

```diff
@@ -4,14 +4,216 @@
+            name="name"
+          >
+        </label>
+
+        <!-- How old are You? -->
+        <label class="modal-field">
+          How old are You?
+
+          <input
+            type="number"
+            name="years"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <!-- Full date of birth -->
+        <label class="modal-field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+
+        <!-- I accept the term of the agreement -->
+        <label class="modal-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <!-- Registration -->
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <!-- E-mail -->
+        <label class="modal-field">
+          E-mail:
+
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required
+            name="email"
+            maxlength="20"
+          >
+        </label>
+
+        <!-- Password -->
+        <label class="modal-field">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <!-- An interesting fact about you! -->
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <!-- Do you love cats? -->
+        <div class="modal-field">
+          <label>
+            Do you love cats?
+
+            <input
+              type="radio"
+              name="cat"
+              value="yes"
+            >
+
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <!-- What is your favorite color? -->
+        <label class="modal-field">
+          What is your favorite color?
+
+          <input
+            type="color"
+            name="color"
+            value="#000000"
+          >
+        </label>
+
+        <!-- What time do you go to bed? -->
+        <label class="modal-field">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="timeToBed"
+            step="2"
+          >
+        </label>
+
+        <!-- What are you favorite brand of cars? -->
+        <label class="modal-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="brandOfCars"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+
+            <option value="Audi">Audi</option>
+
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
```

Comment:
Why do you need so much empty space??

Reply:
I think after one tagBlock we need some space between

Reply:
Not for <option>, it looks weird

Reply:
Ok, I understand it, thanks